### ISSUE TITLE
Python Implementation of EDA

### ISSUE THREAD

**Issue by aqibsaeed** [mateusjoffily/EDA#1]
Do you have any plan to port this implementation to Python? Atleast the code to calculate amplitude and rise time from raw eda values.

**Comment by mateusjoffily** [comment]
Sorry for this very late reply. Just now I saw your comment. No, I don't have any plan to implement it in Python but it would be great to see it done.

**Comment by RobertWan91** [comment]
Do you know any similar package in Python as this implementation of EDA? Thanks.

**Comment by mateusjoffily** [comment]
No, I'm not aware about any other similar package in Pyton. Sorry.
